### DISCUSSION TITLE
USB flashing format (UF2)

### DISCUSSION THREAD

**Discussion by UdayakumarHidakal** [nihalpasham/rustBoot#41] [General]
- Understanding the concepts of `USB flashing format` (uf2).
- collecting information of uf2 python code in git hub and trying to translate into `rust`.

**Comment by jacobrosenthal** [comment]
Might be related to your needs
https://github.com/jacobrosenthal/hf2-rs

**Reply by imrank03** [reply]
Thanks @jacobrosenthal, this would be very helpful.
